### ISSUE TITLE
Value binding issue with null property

### ISSUE THREAD

**Issue by hluz** [ManuelDeLeon/viewmodel#256]
When the value property being bound is null, ViewModel spits the dummy with ```TypeError: Cannot read property 'toString' of null```. Example:

ViewModel:
```
Template.Something.viewmodel({
  fld: null
});
```

Template:
```
<template name="Something">
    <input type="text" {{b 'value: fld'}}>
</template>
```

Would it be possible for ViewModel to guard against it? I can code around it, but it is an hassle if the property comes straight from Mongodb...

**Comment by hluz** [comment]
Sheshh, @ManuelDeLeon ... Y're now fixing issues even before I reported the (just noticed 4.1.5)?!

Luckily (for me) you only fixed half of it, otherwise I would have wasted this issue and would have egg all over my face :-)  I think that line 51 (just above your fix) also needs similar treatment...

**Comment by ManuelDeLeon** [comment]
fixed on 4.1.6

Thanks for the heads up.
